app/models/DepartureUpdate.scala

```diff
@@ -28,23 +28,11 @@ sealed trait DepartureUpdate
 object DepartureUpdate {
 
   implicit val departureUpdateSemigroup: Semigroup[DepartureUpdate] = {
-    case (_: DepartureStatusUpdate, x: DepartureStatusUpdate) => x
-    case (a: DepartureStatusUpdate, m: MessageStatusUpdate)   => CompoundStatusUpdate(a, m)
-    case (_: DepartureStatusUpdate, c: CompoundStatusUpdate)  => c
-
-    case (_: MessageStatusUpdate, x: MessageStatusUpdate)   => x
-    case (m: MessageStatusUpdate, a: DepartureStatusUpdate) => CompoundStatusUpdate(a, m)
-    case (_: MessageStatusUpdate, c: CompoundStatusUpdate)  => c
-
-    case (_: CompoundStatusUpdate, x: CompoundStatusUpdate)  => x
-    case (c: CompoundStatusUpdate, a: DepartureStatusUpdate) => c.copy(departureStatusUpdate = a)
-    case (c: CompoundStatusUpdate, m: MessageStatusUpdate)   => c.copy(messageStatusUpdate = m)
+    case (_: MessageStatusUpdate, x: MessageStatusUpdate) => x
```

Comment:
Can you explain what's happening with this change? I don't love the `DepartureUpdate` thing at all but this would seem to completely break it at least remove any point in its existence. Can `DepartureUpdate` just be removed now?

Reply:
So looking at the old code DepartureUpdate is a sealed trait used to combine updates to the messageStatus and DepartureStatus. We no longer store the DepartureStatus as it's calculated from the messages.

So, yes, I believe that I can just delete DepartureUpdate. I'd missed that. Are you happy with what's happening, and can I go ahead and delete this class (renaming the file to MessageStatusUpdate

Reply:
Yep sure that would be great!

Reply:
Ok, I've updated that. Running the tests also updated the copyright to 2022. But that's fine